### DISCUSSION TITLE
Photoprism unexpectedly splits EXIF keywords by underscore

### DISCUSSION THREAD

**Discussion by karlicoss** [photoprism/photoprism#3712] [Ideas]
I use underscores while tagging photos with digikam/exiftool, e.g. here's exiftool output
```
  "XMP:Categories": "<Categories><Category Assigned=\"1\">for_mom</Category></Categories>",
  "XMP:TagsList": "for_mom",
  "XMP:LastKeywordXMP": "for_mom",
  "XMP:HierarchicalSubject": "for_mom",
  "XMP:CatalogSets": "for_mom",
  "XMP:Subject": "for_mom",
  "IPTC:Keywords": "for_mom",
```
When I import them into Photoprism, it seems to split them by dash, so I end up with two separate Photoprism keywords, "for" and "mom".
Is it on purpose, or just a bug in processing? Thanks!

**Comment by graciousgrey** [comment]
Thanks for sharing your use case! 
Our current search functionality is designed to aggregate keywords from many different sources, many of which contain a lot of noise that would clutter the index.

We plan to additionally offer a [manual tagging system](https://github.com/photoprism/photoprism/issues/1779) that will allow you to organize your pictures with arbitrary tags - that may also include  signs as _, space, numbers and emojis - so that you have complete freedom and your tags are not mixed with automatically extracted metadata.

**Comment by karlicoss** [comment]
Thanks, sounds good! I think another thing that would be nice is to distinguish between ai generated tags and manual/exif tags in the UI and the search. E.g. so it would be possible to search by exif tag "selfie" without finding everything AI tagged determined as selfie.
